src/Table.php

```diff
@@ -384,9 +405,19 @@ public function getRowTemplate()
     {
         $output = [];
         foreach ($this->columns as $name => $column) {
-            $field = $this->model->getElement($name);
 
-            $output[] = $column->getCellTemplate($field);
+            // If multiple formatters are defined, use the first for the header cell
```

Comment:
If data cell has multiple formatters, then all of them should be used here. This is not header cell here !